docs/model_developers/modules.rst

```diff
@@ -90,9 +90,10 @@ The ``setup.cfg`` file defines metadata about the module. The following code sni
   * ``version``: The version of the module. Modules must use semantic versioning.
   * ``license``: The license under which the module is distributed.
 
-* Dependencies to other Inmanta modules and dependencies to external Python libraries can be defined using the
-  ``install_requires`` config option in the ``options`` section of the ``setup.cfg`` file. These version specs use `PEP440
-  syntax <https://www.python.org/dev/peps/pep-0440/#version-specifiers>`_.
+* The ``install_requires`` config option in the ``options`` section of the ``setup.cfg` file defines the dependencies of the
+   module on other Inmanta modules and external Python libraries. These version specs use
+   `PEP440 syntax <https://www.python.org/dev/peps/pep-0440/#version-specifiers>`_. Adding a new module dependency to the module
+   should be done using the ``inmanta module add`` command instead of altering the ``setup.cfg`` file by hand.
```

Comment:
Why? What does this do other than add the `install_requires` and installing the module?

Reply:
Yesterday we had a discussion on Slack regarding when the `inmanta module add` command should  be used. The conclusion was that the documentation should encourage our users to use the command as much as possible instead of applying the changes by hand.

Reply:
Ah, I'd misunderstood that remark. Fine by me.